config/default.json

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
So it will try to read environment variables first defined in custom-environment-variables.json, then it will fallback to local.json if no value is found, then {environment}.json (so if NODE_ENV is set to staging it will use staging.json), then it will check default.json. If the config key is defined nowhere it will throw an error. Every config key should be defined in default.json so we have a clear view of all the configurations even when it is not assigned a value there.

Reply:
Ah very nice sounds good